CompassTests/Shared/TestCompass.swift

```diff
@@ -24,7 +24,7 @@ class TestCompass: XCTestCase {
     let expectation = self.expectationWithDescription("Parse arguments")
     let url = NSURL(string: "compassTests://profile:testUser")!
 
-    Compass.parse(url) { route, arguments in
+    Compass.parse(url) { route, arguments, _ in
```

Comment:
Well that depends on how often that third argument will be used. If it's never used I'd say copy the method and paste it. It feels a bit more weird to me to have it always as a `_`. But that's just a small thing, we can merge this one for now :)